bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/service/DataConverters.java

```diff
@@ -1,86 +0,0 @@
-/**
```

Comment:
The remainder is an empty file, better to delete the file.

Reply:
This file is not in my pull request